DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/reports/accountant-report-export.md

__MultiSafepay does not collect funds for PayPal. Therefore, PayPal transactions do not appear in the accountant export. Make sure that the financial status is set to _Completed_ in your backend. This will then be reflected in the accountant export.__

> If you have any questions regarding this process and/or require any assistance, contact our Support Team at <support@multisafepay.com>

Reviewer: ```suggestion
For support or any questions, email the Support Team at <support@multisafepay.com>
```
